### ISSUE TITLE
Avoiding accessing local net variables that have not yet received a value

### ISSUE THREAD

**Issue by yawlfoundation** [yawlfoundation/yawl#121]
```
I have developed an algorithm that can detect whether it is possible that 
a local variable that has not yet received a value is assigned to a task 
variable. As this leads to a runtime error (the work item cannot be 
started as validation fails) it is desirable that such problems can be 
pointed out to the user at design time. Naturally, only control flow 
aspects are taken into account in this analysis.

 
```

Original issue reported on code.google.com by `arthurte...@gmail.com` on 26 Aug 2008 at 10:17

**Comment by yawlfoundation** [comment]
Original comment by `yawl.mic...@gmail.com` on 28 Aug 2008 at 1:45
* Added labels: **Category-Editor**
* Removed labels: **Category-Component-Editor**

**Comment by yawlfoundation** [comment]
```
The validation error was caused by a editor bug rather than a limitation of 
schema
validation within the engine. But will keep this open as an enhancement request.
```

Original comment by `yawl.mic...@gmail.com` on 28 Aug 2008 at 12:07
* Changed state: **Started**
* Added labels: **Priority-Low**
* Removed labels: **Priority-Medium**

**Comment by yawlfoundation** [comment]
```
This has now been implemented in 2.1beta (although the algorithm used is 
different to that devised by Arthur)
```

Original comment by `yawl.mic...@gmail.com` on 5 May 2010 at 6:01
* Changed state: **Fixed**
